rivertype/river_type.go

```diff
@@ -17,6 +17,15 @@ var ErrNotFound = errors.New("not found")
 // running.
 var ErrJobRunning = errors.New("running jobs cannot be deleted")
 
+// JobArgs is an interface that should be implemented by the arguments to a job.
+// This definition duplicates the JobArgs interface in the river package so that
+// it can be used in other packages without creating a circular dependency.
+type JobArgs interface {
+	// Kind returns a unique string that identifies the type of job. It's used to
+	// determine which worker should work the job.
+	Kind() string
+}
```

Comment:
Needed to use this from `dbunique` so I copied the interface into here. But then I ended up adding an unexported `jobArgs` within that package anyway—maybe that's better than exposing a duplicate in a user-facing package like this?